HLTrigger/Configuration/python/customizeHLTforPatatrack.py

```diff
@@ -54,10 +54,7 @@ def customiseCommon(process):
 
     # Services
 
-    _load_if_missing(process, 'CUDAService', 'HeterogeneousCore.CUDAServices.CUDAService_cfi')
-
-    if 'MessageLogger' in process.__dict__:
-        process.MessageLogger.CUDAService = cms.untracked.PSet()
+    _load_if_missing(process, 'ProcessAcceleratorCUDA', 'HeterogeneousCore.CUDACore.ProcessAcceleratorCUDA_cfi')
```

Comment:
@fwyzard In light of your suggestion https://github.com/cms-sw/cmssw/pull/36699#issuecomment-1016532802, should this actually be replaced with
```suggestion
    process.load('Configuration.StandardSequences.Accelerators_cff')
```
?

Reply:
@fwyzard 
This question is still open - for another PR?

Reply:
@Martin-Grunewald sorry, I missed this.

Yes, that seems a good idea.